startup/81_xpd_map_flyscan.py

```diff
@@ -0,0 +1,472 @@
+"""Plan to run a XRD map "fly-scan" over a large sample."""
+import datetime
+import pprint
+import time as ttime
+import uuid
+
+import numpy as np
+import itertools
+import bluesky.preprocessors as bpp
+import bluesky.plan_stubs as bps
+import bluesky.plans as bp
+from bluesky.utils import short_uid
+
+import bluesky_darkframes
+from ophyd import Signal
+from ophyd.status import Status
+
+
+# vendored, simplified, and made public from bluesky_darkframes
+class SnapshotShell:
+    """
+    Shell to hold snapshots
+    This enables us to hot-swap Snapshot instances in the middle of a Run.
+    We hand this object to the RunEngine, so it sees one consistent
+    instance throughout the Run.
+    """
+
+    def __init__(self):
+        self.__snapshot = None
+
+    def set_snaphsot(self, snapshot):
+        self.__snapshot = snapshot
+
+    def __getattr__(self, key):
+        return getattr(self.__snapshot, key)
+
+
+def _extarct_motor_pos(mtr):
```

Comment:
This looks like a typo to me.
```suggestion
def _extract_motor_pos(mtr):
```